server/models/polls.js

```diff
@@ -0,0 +1,38 @@
+const mongoose = require("mongoose");
+const Schema = mongoose.Schema;
+
+const PollSchema = new Schema({
```

Comment:
Yes agreed, I think for now we don't need it - I'd like to add that once we think about the voters

Reply:
Yes @tgnemecek it's a basic model. Right now, the vote attribute wasn't added because the requirement was not there, as of yet, to accommodate votes attribute at a later stage, the images is a sub-document.